components/admin_console/permission_schemes_settings/permissions_tree/permissions_tree.test.tsx

```diff
@@ -129,11 +130,11 @@ describe('components/admin_console/permission_schemes_settings/permission_tree',
         );
 
         const groups = wrapper.find(PermissionGroup).first().prop('permissions');
```

Comment:
you can assign the type here to avoid repetition

Reply:
Yep, you are right